### DISCUSSION TITLE
Bulk create issues

### DISCUSSION THREAD

**Discussion by drone1** [community/community#11861] [Projects and Issues]
I can select multiple items in a Board view by clicking + shift (or ctrl), but there seems to be no way to bulk-convert all items to an issue. 

I'm adding many tasks here so this makes using Projects pretty cumbersome for my use case unfortunately.

**Comment by Lippiece** [comment]
Edit: apologies, I've misunderstood the exact topic. However, you may still find the script useful.

Since this isn't answered yet, and I've stumbled upon this upon searching the solution myself, here's a fish script:

<details>

```fish
#!/usr/bin/fish

set repo $argv[1] 
set issues $argv[2]

set delimiter '|'

set tmpfile (mktemp)
cp $issues $tmpfile

while read -l line < $tmpfile
    set title (string split -m 1 -- $delimiter $line)[1] 
    set desc (string split -m 1 -- $delimiter $line)[2]

    gh issue create --title "$title" --body "$desc" --repo $repo
    
    sed -i 1d $tmpfile
end 
```
</details>

It accepts text files in this format:
<details>

```
Add docs for X feature|We need to document how X feature works
Fix bug in Y|Y is currently broken due to Z
```

</details>
